ansible_collections/arista/avd/molecule/dhcp_provisioning/molecule.yml

```diff
@@ -1,29 +1,24 @@
 ---
 
 scenario:
-  create_sequence:
-    - dependency
-    - prepare
   converge_sequence:
-    - dependency
     - create
-    - prepare
     - converge
   test_sequence:
-    - dependency
     - cleanup
     - destroy
     - syntax
     - create
-    - prepare
     - converge
   cleanup_sequence:
     - destroy
 dependency:
   name: galaxy
   enabled: false
 driver:
-  name: docker
+  name: default
```

Comment:
Why do we need these changes to the molecule scenario?

Reply:
it is more that we don't need docker for offline mode and so we are triggering extra hooks in the scenario that are not needed. The other scenario `dhcp_configruation` (and yes we can argue that the names are confusing) for online mode still requires docker containers.